src/bootstrap/install.rs

```diff
@@ -165,10 +165,15 @@ install!((self, builder, _config),
         }
     };
     RustAnalyzer, "rust-analyzer", Self::should_build(_config), only_hosts: true, {
-        let tarball = builder
-            .ensure(dist::RustAnalyzer { compiler: self.compiler, target: self.target })
-            .expect("missing rust-analyzer");
-        install_sh(builder, "rust-analyzer", self.compiler.stage, Some(self.target), &tarball);
+        if let Some(tarball) =
+            builder.ensure(dist::RustAnalyzer { compiler: self.compiler, target: self.target })
+        {
+            install_sh(builder, "rust-analyzer", self.compiler.stage, Some(self.target), &tarball);
```

Comment:
Hm, I think while we're at it we should do this for all of the steps in install that expect(...) the dist step. My guess is there's a similar failure caused by setting docs = false waiting to happen, for example.

Reply:
Yea, good point. I surveyed the other expects and they seem safe to me. I added some comments to that effect. Though I guess that is somewhat a style issue, if you want to guard against future changes, I can add some additional skips.